app/models/user.rb

```diff
@@ -30,6 +30,25 @@ def self.search(input)
     all
   end
 
+  # Generates a token digest.
+  def self.generate_digest(token)
+    Digest::SHA2.hexdigest(token)
+  end
```

Comment:
For now this is fine, though we may want to move this to a service in the future

Reply:
Yup, we may generate more secure tokens and crypto sign there digest